app/services/statistics_calculator.rb

```diff
@@ -1,34 +1,69 @@
 class StatisticsCalculator
 
+    module Constants
+        NUMBER_OF_QUESTIONS = 10
+        FULL_PERCENTAGE = 100
+    end
+    
+    module Score
+        A = 90
+        B = 75
+        C = 60
+    end
+    
     class << self 
 
         def success_per_question(results)
-            statistics = Array.new;
-            questions = Array.new(10,0);
-            
+            questions = Array.new(Constants::NUMBER_OF_QUESTIONS, 0)  
             results.each {|result| count_correct_answers_in_result(result.answers, questions)}
-            claculate_correct_answers_percentage(questions, results.length(), statistics)
-            statistics
+            questions.map.with_index {|question, index| calculate_correct_answers_percentage(question, index, results.length())}
```

Comment:
minor - convention is not to use parenthesis when function has no params
```suggestion
            questions.map.with_index {|question, index| calculate_correct_answers_percentage(question, index, results.length)}
```